LKSH460/iocBoot/iocLKSH460-IOC-01/st-common.cmd

```diff
@@ -0,0 +1,44 @@
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(LKSH460)/Lakeshore_460Sup"
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(LKSH460)/data"
+epicsEnvSet "DEVICE" "L0"
+
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+cd "${TOP}"
+
+# For dev sim devices
+$(IFDEVSIM) drvAsynIPPortConfigure("L0", "localhost:$(EMULATOR_PORT=57677)")
+
+## Register all support components
+dbLoadDatabase "dbd/LKSH460-IOC-01.dbd"
+LKSH460_IOC_01_registerRecordDeviceDriver pdbbase
```

Comment:
This is duplicated in both `st.cmd` and here, which causes the following errors in the IOC log:

```
...
LKSH460_IOC_01_registerRecordDeviceDriver pdbbase
Warning: Registration already done.
registryFunctionRefAdd: could not register arrayTest_init
registryFunctionRefAdd: could not register asSubInit
registryFunctionRefAdd: could not register configMenuList_init
registryFunctionRefAdd: could not register interp_init
registryFunctionRefAdd: could not register initSubAve
Warning: iocshRegisterVariable redefining CASDEBUG.
Warning: iocshRegisterVariable redefining aCalcAsyncThreshold.
Warning: iocshRegisterVariable redefining aCalcLoopMax.
Warning: iocshRegisterVariable redefining aCalcPerformDebug.
Warning: iocshRegisterVariable redefining aCalcPostfixDebug.
Warning: iocshRegisterVariable redefining aCalcoutRecordDebug.
Warning: iocshRegisterVariable redefining arrayTestDebug.
Warning: iocshRegisterVariable redefining asCaDebug.
Warning: iocshRegisterVariable redefining atExitDebug.
Warning: iocshRegisterVariable redefining boHIGHlimit.
Warning: iocshRegisterVariable redefining boHIGHprecision.
Warning: iocshRegisterVariable redefining calcoutODLYlimit.
Warning: iocshRegisterVariable redefining calcoutODLYprecision.
Warning: iocshRegisterVariable redefining callbackParallelThreadsDefault.
Warning: iocshRegisterVariable redefining configMenuDebug.
Warning: iocshRegisterVariable redefining dbBptNotMonotonic.
Warning: iocshRegisterVariable redefining dbQuietMacroWarnings.
Warning: iocshRegisterVariable redefining dbRecordsAbcSorted.
Warning: iocshRegisterVariable redefining dbRecordsOnceOnly.
Warning: iocshRegisterVariable redefining dbTemplateMaxVars.
Warning: iocshRegisterVariable redefining dbThreadRealtimeLock.
Warning: iocshRegisterVariable redefining debugSubAve.
Warning: iocshRegisterVariable redefining devaCalcoutSoftDebug.
Warning: iocshRegisterVariable redefining devsCalcoutSoftDebug.
Warning: iocshRegisterVariable redefining histogramSDELprecision.
Warning: iocshRegisterVariable redefining interpDebug.
Warning: iocshRegisterVariable redefining sCalcLoopMax.
Warning: iocshRegisterVariable redefining sCalcPerformDebug.
Warning: iocshRegisterVariable redefining sCalcPostfixDebug.
Warning: iocshRegisterVariable redefining sCalcStackHW.
Warning: iocshRegisterVariable redefining sCalcStackLW.
Warning: iocshRegisterVariable redefining sCalcoutRecordDebug.
Warning: iocshRegisterVariable redefining save_restoreDatedBackupFiles.
Warning: iocshRegisterVariable redefining save_restoreDebug.
Warning: iocshRegisterVariable redefining save_restoreIncompleteSetsOk.
Warning: iocshRegisterVariable redefining save_restoreNumSeqFiles.
Warning: iocshRegisterVariable redefining save_restoreRemountThreshold.
Warning: iocshRegisterVariable redefining save_restoreSeqPeriodInSeconds.
Warning: iocshRegisterVariable redefining seqDLYlimit.
Warning: iocshRegisterVariable redefining seqDLYprecision.
Warning: iocshRegisterVariable redefining sseqRecDebug.
Warning: iocshRegisterVariable redefining streamDebug.
Warning: iocshRegisterVariable redefining swaitRecordDebug.
Warning: iocshRegisterVariable redefining transformRecordDebug.
...
```

Reply:
Removed duplication